site/docs/community.md

```diff
@@ -73,3 +73,12 @@ Iceberg has four mailing lists:
     - [Archive](https://lists.apache.org/list.html?issues@iceberg.apache.org)
 * **Private**: <private@iceberg.apache.org> -- private list for the PMC to discuss sensitive issues related to the health of the project
     - [Archive](https://lists.apache.org/list.html?private@iceberg.apache.org)
+
+
+## Setting up IDE and Code Style
+
+### Configuring Code Formatter for IntelliJ IDEA
+
+Navigate to **File > Settings > Editor > Code Style > Java**. Click on the gear wheel and select **Import Scheme** to import IntelliJ IDEA XML code style settings.
```

Comment:
This differs from systems. It is "IntelliJ IDEA > Preferences > Editor > Code Style ..." in MacOS. We may just mention that "Go to Editor | Code Style".

Reply:
updated the description